src/app/shared/models/my-account-path.ts

```diff
@@ -0,0 +1 @@
+export const MyAccountPath: string = 'moje-konto';
```

Comment:
Suggested refactor of a file `help-type-path.enum` to `core-path.enum`

with changes:

```
export enum CorePath {
  FindHelp = 'znajdz-pomoc',
  GiveHelp = 'dodaj-ogloszenie',
  MyAccount = 'moje-konto'
}
```

Reply:
done